### PR TITLE
Triggered Kotsadm version update 1.50.0-beta.0

### PR DESCRIPTION
**Pull request by replicatedbot**
Automated changes by the [auto-kotsadm-update](https://github.com/replicatedhq/kURL/blob/master/.github/workflows/update-kotsadm.yaml) GitHub action

### CODE REVIEW

File: addons/kotsadm/alpha/Manifest [hunk 1]
```diff
@@ -1,9 +1,8 @@
 image kotsadm-migrations docker.io/kotsadm/kotsadm-migrations:alpha
-image kotsadm-operator docker.io/kotsadm/kotsadm-operator:alpha
 image kotsadm-minio minio/minio:RELEASE.2021-07-27T02-40-15Z
 image kotsadm docker.io/kotsadm/kotsadm:alpha
 image kurl-proxy docker.io/kotsadm/kurl-proxy:alpha
 image postgres postgres:10.17-alpine
 image dex docker.io/kotsadm/dex:v2.28.1
 
-asset kots.tar.gz https://github.com/replicatedhq/kots/releases/download/v1.49.0/kots_linux_amd64.tar.gz
+asset kots.tar.gz https://github.com/replicatedhq/kots/releases/download/1.50.0-beta.0/kots_linux_amd64.tar.gz
```
Comment (DanStough) [inline addons/kotsadm/alpha/Manifest:8]: This is wrong (two places) and a problem with the generate script for betas. See: https://github.com/replicatedhq/kURL/pull/2014/commits/39e4a578fb774e3a5c5c91d0d31a8ef24af4c89a#diff-39f99d92c381600cd96e3ef4ea1204b328b76cb8a666862dd9c08a4c4d499775R51-R52

File: addons/kotsadm/alpha/install.sh [hunk 2]
```diff
@@ -365,7 +370,7 @@ function kotsadm_cli() {
     fi
     if [ ! -f "$src/assets/kots.tar.gz" ] && [ "$AIRGAP" != "1" ]; then
         mkdir -p "$src/assets"
-        curl -L "https://github.com/replicatedhq/kots/releases/download/v1.49.0/kots_linux_amd64.tar.gz" > "$src/assets/kots.tar.gz"
+        curl -L "https://github.com/replicatedhq/kots/releases/download/1.50.0-beta.0/kots_linux_amd64.tar.gz" > "$src/assets/kots.tar.gz"
```
Comment (DanStough) [inline addons/kotsadm/alpha/install.sh:373]: also wrong here.

**Review by DanStough — CHANGES_REQUESTED** [review]

File: addons/kotsadm/alpha/install.sh (hunk 2 above)
Comment (sgalsaleh) [inline addons/kotsadm/alpha/install.sh:373]:
```suggestion
        curl -L "https://github.com/replicatedhq/kots/releases/download/v1.50.0-beta.0/kots_linux_amd64.tar.gz" > "$src/assets/kots.tar.gz"
```

File: addons/kotsadm/alpha/Manifest (hunk 1 above)
Comment (sgalsaleh) [inline addons/kotsadm/alpha/Manifest:8]:
```suggestion
asset kots.tar.gz https://github.com/replicatedhq/kots/releases/download/v1.50.0-beta.0/kots_linux_amd64.tar.gz
```

**Review by DanStough — APPROVED** [review]

**Pull request merged**